changelog.d/20250630_144536_yadudoc1729_get_endpoints_with_role.rst

```diff
@@ -0,0 +1,4 @@
+Added
+-----
+
+- Updated `ComputeClientV2.get_endpoints` with a new `role` kwarg. (:pr:`1238`)
```

Comment:
pre-commit caught this and I missed it (RST vs Markdown backtick styles, which render differently):

```suggestion
- Updated ``ComputeClientV2.get_endpoints`` with a new ``role`` kwarg. (:pr:`1238`)
```

Reply:
This is addressed.